Lib/shutil.py

```diff
@@ -1030,8 +1030,10 @@ def make_archive(base_name, format, root_dir=None, base_dir=None, verbose=0,
     uses the current owner and group.
     """
```

Comment:
not sure what the rules are re-auditing, would it be better to do:
```
base_name = base_name ?? os.fsdecode(base_name)
root_dir = root_dir ?? os.fsdecode(root_dir)
base_dir = base_dir ?? os.fsdecode(base_dir)
```
before the audit call or after?

Reply:
What do you mean by `??` ?

Reply:
https://peps.python.org/pep-0505/ syntax